lib/app_web/controllers/page_controller.ex

```diff
@@ -12,25 +12,24 @@ defmodule AppWeb.PageController do
   def index(conn, _params) do
     url_oauth_google = ElixirAuthGoogle.generate_oauth_url(conn)
     changeset = Person.changeset_registration(%Person{}, %{})
-    render(conn, "index.html",
-      [url_oauth_google: url_oauth_google, changeset: changeset])
+    render(conn, "index.html", url_oauth_google: url_oauth_google, changeset: changeset)
```

Comment:
This line is now 88 characters long ... is there a way of telling `mix format` to